solution/ui/regulations/eregs-vite/src/components/resources/SearchGovResults.vue

```diff
@@ -0,0 +1,208 @@
+<script setup>
+import {
+    formatDate,
+    locationLabel,
+    locationUrl,
+} from "../../utilities/filters";
+
+import ResultsItem from "@/components/search/ResultsItem.vue";
+import SupplementalContentObject from "eregsComponentLib/src/components/SupplementalContentObject.vue";
+
+defineProps({
+    base: {
+        type: String,
+        required: true,
+    },
+    partsLastUpdated: {
+        type: Object,
+        required: true,
+    },
+    partsList: {
+        type: Array,
+        required: true,
+    },
+    results: {
+        type: Array,
+        default: () => [],
+    },
+});
+
+const openChar = ""; // &#xe000;
+const closeChar = ""; // &#xe001;
+const highlightClass = "search-highlight";
+
+const needsBar = (item) => item.date && item.name;
+
+/**
+ * @param string {string} - string containing substrings surrounded by special characters
+ * @param startChar {string} - character that indicates the start of a substring to get
+ * @param stopChar {string} - character that indicates the end of a substring to get
+ * @returns {Array{string}} - Array of unique substrings
+ */
+const getHighlightTerms = (string, startChar, stopChar) => {
```

Comment:
nicely done here, took me a second to wrap my head around it

Reply:
I only got halfway to nirvana.  Next time I do something like this, I'm going to get a `pipe` operator working: https://jrsinclair.com/articles/2022/javascript-function-composition-whats-the-big-deal/

Or maybe the JS pipeline operator (`|>`) will be released with ES2023 by then.  
https://github.com/tc39/proposal-pipeline-operator